src/utils.js

```diff
@@ -76,6 +78,8 @@ export const generateProjectConfig = (projectId, name) => ({
   name: name,
   id: projectId,
   secret: generateId(),
+  aesKey: encodeBase64,
```

Comment:
You should call the function here so that a unique value is generated everytime a project is created. As of now what you have done will cretae one encodebase64 value globally and use that for each project being created in that session.